lib/privs.c

```diff
@@ -25,11 +25,16 @@
 #include "privs.h"
 #include "memory.h"
 #include "lib_errors.h"
-
-#ifdef HAVE_CAPABILITIES
+#include "lib/queue.h"
 
 DEFINE_MTYPE_STATIC(LIB, PRIVS, "Privilege information")
 
+#ifndef HAVE_CAPABILITIES
+static bool privs_per_process = true;
```

Comment:
This should be `const` as well, I think, and perhaps it'd be useful to group all the possibilities for defining this variable next to each other for clarity